install/prerequisites.adoc

```diff
@@ -724,23 +726,61 @@ It depends on the target host of the deployment and uses the computed value of
     --dport 1936 -j ACCEPT
 ----
 
-.Aggregated Logging
+.Aggregated Logging and Metrics
 [cols='2,1,8']
 |===
 | *9200*
 |TCP
-|For Elasticsearch API use. Required to be internally open on any infrastructure
+| For Elasticsearch API use. Required to be internally open on any infrastructure
 nodes so Kibana is able to retrieve logs for display. It can be externally
 opened for direct access to Elasticsearch by means of a route. The route can be
 created using `oc expose`.
 
 | *9300*
 |TCP
-|For Elasticsearch inter-cluster use. Required to be internally open on any
+| For Elasticsearch inter-cluster use. Required to be internally open on any
 infrastructure node so the members of the Elasticsearch cluster might communicate
 with each other.
+
+| *9090*
+| TCP 
+| For Prometheus web console use.
```

Comment:
9090 is also used to reload the prometheus configuration, so should be
For Prometheus API and web console use.

Reply:
@juzhao I made the change as suggested.